Aydin/backend/src/lecture/lecture_entity.ts

```diff
@@ -8,18 +8,19 @@ import {
 } from 'typeorm';
 import {Course} from '../course/course_entity';
 import {Student} from '../student/student_entity';
+
 @Entity()
 export class Lecture {
   @PrimaryColumn()
-  lectureName: string;
+  lecture_name!: string;
```

Comment:
does this variable name have to match the column name in the database? again, i'm asking because i don't know TypeORM, though generally we use camel case for all field names in JS

Reply:
yes it have to match the column in database.I used to write it in camel case but i read online that usually it should be lecutre_name, because typreorm transform the column name to small case so lectureName would be lecturename.